update_python_spec.lisp

```diff
@@ -1,11 +1,21 @@
 #!/usr/bin/env sbcl --script
 (load "~/.sbclrc")
 (ql:quickload :rpcq)
-(rpcq::clear-messages)
+
+; messages
+(rpcq::clear-messages :messages)
 (load "src/messages.lisp")
 (with-open-file (f "rpcq/messages.py"
                    :direction ':output
-                   :if-exists ':supersede) 
-  (rpcq::python-message-spec f)
-  (write-line "Wrote new message spec."))
+                   :if-exists ':supersede)
+  (rpcq::python-message-spec f (gethash :messages rpcq::*messages*))
+  (write-line "Wrote new messages.py"))
 
+; core-messages
+(rpcq::clear-messages :core-messages)
+(load "src/core-messages.lisp")
+(with-open-file (f "rpcq/core-messages.py"
+                   :direction ':output
```

Comment:
we should add
```
:if-does-not-exist ':create
```

Reply:
So it already did that without me adding this?